client/src/app.jsx

```diff
@@ -3,34 +3,55 @@ import ReactDOM from 'react-dom';
 import $ from 'jquery';
 import ReviewList from './ReviewList.jsx';
 import helpers from './helpers.js';
+//let nodeEnv = process.env.NODE_ENV;
+// const dotenv = require('dotenv');
+// dotenv();
 
 class Reviews extends React.Component {
   constructor(props) {
     super();
     this.state = {
       list: [],
+      ownerInfo: {},
       sortOption: 'Most Popular'
     }
     this.sortReviews = this.sortReviews.bind(this);
   }
 
   componentDidMount() {
-    let search = window.location.search
+    let search = window.location.search;
+    console.log('search', search);
+    let host = window.location.host;
     let index = search.indexOf('=') + 1;
     let id = search.slice(index);
+    //let url = `http://${host}/reviews`;
+    // let url = '';
+
+    // if (nodeEnv === 'development' || !nodeEnv) {
+    //   url = helpers.urls.development;
+    // } else {
+    //   url = helpers.urls.production;
+    // }
+    // console.log('url', url, 'env', nodeEnv)
 
     $.ajax({
-      url: `http://localhost:3001/reviews`,
+      url: 'http://ec2-54-193-152-3.us-west-1.compute.amazonaws.com/reviews',
```

Comment:
i don't think you'll need to do this since you are fully deployed, but you may want to add this to a separate config file that you import in so that you can change the number url easily, i.e. not hunt it down in your code.